packages/fluentui/react-northstar/src/utils/renderComponent.tsx

```diff
@@ -80,7 +80,8 @@ const renderComponent = <P extends {}>(
     className,
     disableAnimations: context.disableAnimations || false,
     primaryDisplayName: displayName,
-    props: stateAndProps,
+    componentProps: stateAndProps,
+    stylesProps: stateAndProps,
     renderer: context.renderer || noopRenderer,
```

Comment:
Cache usage is blocked for class components so there is no need to separate them somehow.